test/unittest/layers/unittest_layers_attention.cpp

```diff
@@ -16,9 +16,10 @@
 #include <attention_layer.h>
 #include <layers_common_tests.h>
 
-auto semantic_attention =
-  LayerSemanticsParamType(nntrainer::createLayer<nntrainer::AttentionLayer>,
-                          nntrainer::AttentionLayer::type, {}, 0, false, 2);
+auto semantic_attention = LayerSemanticsParamType(
+  nntrainer::createLayer<nntrainer::AttentionLayer>,
+  nntrainer::AttentionLayer::type, {},
+  LayerCreateSetPropertyOptions::AVAILABLE_FROM_APP_CONTEXT, false, 2);
```

Comment:
I couldn't find any reason to replace 0 to 1 (LayerCreateSetPropertyOptions::AVAILABLE_FROM_APP_CONTEXT).
Could you check this?

Reply:
Previous option value was 0 which makes every layers to create object(layers_dependent_common_test.cpp:24) and if it throws then it register the layer to app context. In other words create object was used to test layer is already register or not.
But we already know which layer is registered by default so by giving the option LayerCreateSetPropertyOptions::AVAILABLE_FROM_APP_CONTEXT it can be skipped.

Reply:
So, we can check it's defaultly registered or not..
Thanks for explaining.